src/Search_Replace_Command.php

```diff
@@ -165,10 +168,14 @@ public function __invoke( $args, $assoc_args ) {
 		$total           = 0;
 		$report          = array();
 		$this->dry_run         = \WP_CLI\Utils\get_flag_value( $assoc_args, 'dry-run' );
-		$php_only        = \WP_CLI\Utils\get_flag_value( $assoc_args, 'precise' );
+		$php_only              = \WP_CLI\Utils\get_flag_value( $assoc_args, 'precise' );
```

Comment:
Not a fan of manually aligning assignments that happen to follow each other (it's so awkward to maintain and fundamentally random - when do you start/stop?!) so personally would just remove but ignore me.